### PR TITLE
feat: added stylesheet for grid layout

### PR DESCRIPTION
**Pull request by rdshinde**
Added stylesheet for simplified grid layout.

https://user-images.githubusercontent.com/67017632/153703989-b5f3e89a-1c2b-4b9b-a94a-9706ebcd8f38.mov

### CODE REVIEW

File: components/grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  grid-template-columns: repeat(auto-fit, minmax(32%, auto));
+}
+.col-4 {
+  grid-template-columns: repeat(auto-fit, minmax(24%, auto));
+}
+
+/* Breakpoints For Columns */
+@media screen and (max-width: 1024px) {
+  .col-2 {
+    display: grid;
```
Comment (sadathshariff) [inline components/grid/grid.css]: Just a suggestion to make display: grid as utility class as you are using it multiple places

Comment (rdshinde) [reply components/grid/grid.css]: Actually I don't need that in col classes. Thanks for pointing out.

Comment (sadathshariff) [reply components/grid/grid.css]: okay got it

**Review by sadathshariff — APPROVED** [review]

**Review by Radheshyam1234 — COMMENTED** [review]
Looks good

**Pull request merged**